src/lib.rs

```diff
@@ -402,7 +397,7 @@ pub enum Error {
     /// The data or human-readable part is too long or too short
     InvalidLength,
     /// Some part of the string contains an invalid character
-    InvalidChar(u8),
+    InvalidChar(char),
```

Comment:
Is this enum variant change a breaking change that needs a major version bump?

Reply:
Good catch, sorry, I will have to bump the version to `0.5.0`. Would you generally agree that it's better to process unicode strings char-wise instead of their UTF-8 representation byte-wise? Because this change is what made this API break necessary.